install.sh

```diff
@@ -7,4 +7,4 @@ git submodule sync
 git submodule update
 
 # for react-native
-npm install react-native@0.35.0 react@15.3.1 --silent
```

Comment:
Looking at https://github.com/facebook/react-native/blob/v0.43.0/package.json#L129,
I think it should be react@16.0.0-alpha.6